src/raft/uv.c

```diff
@@ -108,23 +107,13 @@ static int uvInit(struct raft_io *io, raft_id id, const char *address)
 		return rv;
 	}
 
-	rv = uvMetadataLoad(uv->dir, &metadata, io->errmsg);
-	if (rv != 0) {
-		return rv;
-	}
-	uv->metadata = metadata;
-
```

Comment:
Why are we moving this and the timer to `_start` and `_load`?

Reply:
uvMetadataLoad checks the format_version field to detect malformed metadata files. The correct format version isn't known until load or bootstrap time, so I moved the metadata loading later on. But I'm definitely less than 100% certain that this is the right way to do it. Maybe it would be a better idea to just modify raft_uv_init to accept the desired format version as an additional argument?